vehicle_diag_smach/rule_based_models/Saugrohrdrucksensor.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# @author Tim Bohne
+
+
+import numpy as np
+
+from vehicle_diag_smach.interfaces.rule_based_model import RuleBasedModel
+
+
+class Saugrohrdrucksensor(RuleBasedModel):
+    """
+    Rule-based model for "Saugrohrdrucksensor" channels.
+    """
+
+    def __init__(self):
+        pass
+
+    def predict(self, input_signal: np.ndarray, chan_name: str = "") -> bool:
+        """
+        Predicts whether the input signal comprises an anomaly.
+
+        :param input_signal: input signal to be classified
+        :param chan_name: optional name of the channel
+        :return: true -> anomaly, false -> no anomaly
+        """
+        if chan_name == "Signalleitung (Druck) des Saugrohrdrucksensors":
+            return True
+        elif chan_name == "Signalleitung (Temperatur) des Saugrohrdrucksensors":
+            return True
```

Comment:
Is there a reason why for some specific channels true is returned? Does it have to do with what result you would expect given the real rules and the test signals, or were the channels just selected randomly?

Reply:
At the moment this is just randomly assigned for testing purposes. As discussed, you'll implement the real version of this (cf. #32 ).